continuous-integration/styleci/configurations/general-php.md

```diff
@@ -0,0 +1,49 @@
+# StyleCI - General PHP
+
+The following is a recommended configuration for general PHP projects which may or may not use a framework. If you are
+using Laravel please use the alternative configuration [here](laravel.md). This configuration uses the `psr2` 
+preset with risky heuristics enabled. It is also tailored to suit our internal PHP & JavaScript guidelines.
+
+```yaml
+php:
+  preset: psr2
+  
+  risky: true
+
+  enabled:
+    - concat_with_spaces
```

Comment:
I feel like we should leave this out (as default) until consensus is reached in #74.

Reply:
> I feel like we should leave this out (as default) until consensus is reached in #74.

I left it in as an assumption that whitespaces will win, I had no plans on merging it in until that issue has been decided on.

Reply:
Awesome. :+1: